tests/tests/atmwav_control_noaero_p8

```diff
@@ -12,38 +12,38 @@ export LIST_FILES="sfcf000.nc \
                    sfcf012.nc \
```

Comment:
@SadeghTabas-NOAA Are you changing this test to run for only 3 hours? I don't see you changing the names of the forecast files (ie, atm12) or the coupler restart file (cpl.r) or wave output (*.ww3) files?